features/crm/Calendar.md

```diff
@@ -0,0 +1,75 @@
+
+Its **user-friendly design** offers quick and easy event management and visual orientation throughout the set of appointments.
+
+Virtually all shared calendars can show appointments within the company or group they are shared with.
+ERP.net Calendar, however, is much more than just a typical calendar.
+
+*Most ERP systems* encounter difficulties with centralized event and resource management when a single business has registered more than one legal entity and operates these entities separately from a legal perspective. 
+
+Unlike many competitive ERP systems, the Calendar of ERP.net shares event information, in real time, not only with the staff members of a single legal entity but also with all businesses and parties within a common business structure or business franchise.
+
+For **users** this means they can **create, edit, visualize, and work** on common appointments, even when these events concern participants taking roles in separate legal entities that are part of holdings and vast corporate business structures spanning multiple geographical locations and international markets.
+
+> [!NOTE]
+> ERP.net Calendar shares event information not only within the selected active single legal entity but also with all businesses and parties shared within a common database. 
+> This feature strongly facilitates time and resource planning, providing precise and timely scheduling of each team's tasks.
+
+## Calendar Views And Navigation
+
+The six Calendar views allow for more appropriate representation of the appointments according to the specific period of the schedule:
+
+* Day View - visualizes appointments throughout the **selected day**
+* Work Week View - visualizes appointments throughout the **selected work week** (Monday to Friday for Europe and UK)
+* Week View - visualizes appointments throughout the **selected 7-day week**
+* Month View - visualizes appointments throughout the **selected month**
+* Timeline View - visualizes appointments on a **timeline** with days iterating on a horizontal axis
+* Gantt View - visualizes appointments as a **Gantt chart**, which is a useful feature for project management and task completion tracking
+
+The **Back** and **Forward** buttons in the **Navigation** tab of the ribbon provide convenient one-click switching between **adjacent time periods** in the currently set **Calendar view**.
+
+## Calendar Appointments And Types
+
+Every party involved has their own custom necessities and requirements. 
+ERP.net Calendar provides a **diversity of appointment types** to meet all needs. 
+Hence, every new appointment can be created as one of the following:
+
+* Sick Leave
+* Personal Engagement
+* Licenses And Client Subscriptions
+* Paid / Unpaid Leave
+* Meeting
+* Subscription Cancellation
+* Proposition
+* Potential Client Inquiry
+* Task
+* Demonstration
+* Message
+* Project Charter
+* Publication
+* Advertising Activity
+* Status Report
+
```

Comment:
What are these? If this is user-defined, this should be specified.

Reply:
Regarding remarks: 

What are these? If this is user-defined, this should be specified. - Document types are now described with more clarity. Description of what is system-based and predefined, and what is user-configurable or vendor-configurable.